### ISSUE TITLE
Move everything in Docs out of Getting Started

### ISSUE THREAD

**Issue by oxinabox** [JuliaPlots/Plots.jl#1369]
Getting Started contains way too much.
Most projects have just Install instructions and some basic example/tutorial.

It actually took me over a year to realize that there was an attributes page, because I never read the getting started, since i got started I think before their were docs.

I suggest the following structure for the side bar:

 - Home
 - Learning
 - Contributing 
 - Ecosystem 
 - Getting Started
	 - Installation
	 - Basics
	 - Tutorial
 - Manual
	 - Input Data
	 - Output
	 - Attributes
	 - Backends
	 - Supported Attributes
	 - Colors
	 - Animations		 
 - Advanced Topics
	 - Internals
	 - Layouts
	 - Recipes
	 - Plots v0.7
 - Examples
	 - PyPlot
	 - GR
	 - PlotlyJS
	 - PGFPlots
	 - UnicodePlots
	 - InspectDR

In short it is basically move Tutorial into "Getting Started",
and move everything else into "Manual",
and reorder things.

**Comment by mkborregaard** [comment]
I agree, except that I think we should delete Plots v0.7, move Layouts and Recipes to Manual, and just keep Internals and pipeline under "Advanced Topics". (I also want the Ecosystem page to change, and StatPlots and PlotRecipes readmes to be moved to separate tabs). 
This should really be an issue on PlotDocs, though.
